Makefile

```diff
@@ -11,8 +11,13 @@ default : install
 build : test
 	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" .
 
-install : test
+install : build 
 	CGO_ENABLED=0 go install -trimpath -ldflags "$(LDFLAGS)" github.com/gittuf/gittuf
 
 test :
 	go test -v ./...
+
+.PHONY: fmt
+
+fmt:
+	@find . -type d \( -name third_party -prune \) -o -name '*.go' -print | grep -v /third_party/ | xargs -n 1 dirname | sort -u | xargs go fmt
```

Comment:
After #145 is closed, this is going to be a lot simpler right? It's unfortunate that `go fmt` doesn't include an option to ignore certain paths / packages. I'm wary of how portable this is on, say, Windows. WDYT of simplifying this to just `go fmt ./...` once the third_party dir is removed? That should be fairly soon.

Reply:
Fixed